README.md

```diff
@@ -1,5 +1,5 @@
 <div align="center">
-  <img width="600" alt="onedl-mmengine" src="https://github.com/user-attachments/assets/23fd3a03-970b-4886-bd2e-becac66de7b4" />
+  <img width="600" alt="onedl-mmengine" src="https://raw.githubusercontent.com/VBTI-development/onedl-mmengine/main/docs/en/_static/image/mmengine-logo.png" />
```

Comment:
```suggestion
  <img width="600" alt="onedl-mmengine" src="https://mmassets.onedl.ai/assets/mmengine-logo.png" />
```

Reply:
I added the image to a public bucket. This should be a better solution then hosting assets on github.